tests/build.gradle

```diff
@@ -149,6 +149,8 @@ dependencies {
     compile project(':core:invoker')
     compile project(':tools:admin')
 
+    compile group: 'io.opentracing', name: 'opentracing-mock', version: '0.31.0'
```

Comment:
Same as above, please move this line up to the other dependencies and adjust the format to what we used everywhere.